src/SignalR/server/Core/src/DefaultHubFilter.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public class DefaultHubFilter : IHubFilter
```

Comment:
We can get rid of this one since I fallback to our old code when no filters are added.